### ISSUE TITLE
Mandatory edu guide requirements

### ISSUE THREAD

**Issue by daviddao** [biojs/edu#41]
Hey guys,
I noticed that many of our students/readers have ERROR messages because of out-dated versions of npm, slush or other dependencies. Because the whole BioJS ecosystem depends on modules, we should tell our readers (at least for our internal biojs modules), which version we used to create this guide/tutorial. 

I would propose to create an automatic section in each tutorial called: biojs dependencies (including version number)

Cheers,
David

**Comment by saketkc** [comment]
+1

**Comment by greenify** [comment]
> I noticed that many of our students/readers have ERROR messages because of out-dated versions of npm, slush or other dependencies.

I disagree. The error happened because of Ubuntu 14.04 LTS which ships npm 1.4 by default  NOT because our internal biojs modules.
Slush-biojs should throw an error when used with npm < 2, that is the easiest solution!

> I would propose to create an automatic section in each tutorial called: biojs dependencies (including version number)

Hmm what would be the benefit of that? We always want to use the latest version of `slush` or `sniper` and npm installs by default the latest version.
However we could highlight more that one should update `slush` and `sniper` from time to time.

**Comment by greenify** [comment]
> Slush-biojs should throw an error when used with npm < 2, that is the easiest solution!

Done with https://github.com/biojs/slush-biojs/commit/22d2d13a5a0705814aa6156527bd2fd23dd4bbe2
